mods/ra/weapons/missiles.yaml

```diff
@@ -75,93 +36,83 @@ Dragon:
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
+^AntiGroundMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Ground, Water
+
+^AntiAirMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Air
+
+Maverick:
+	Inherits: ^AntiGroundMissile
+	ReloadDelay: 30
+	Range: 9c0
+	MinRange: 3c0
+	Report: missile7.aud
+	Burst: 2
+	BurstDelay: 7
+	Projectile: Missile
+		Speed: 256
+		Inaccuracy: 512
+		CruiseAltitude: 2c0
+		RangeLimit: 14c410
+	Warhead@1Dam: SpreadDamage
+		Damage: 70
+		Versus:
+			None: 30
+			Wood: 90
+			Light: 90
+			Heavy: 100
+			Concrete: 100
+
+Dragon:
+	Inherits: ^AntiGroundMissile
+
 HellfireAG:
+	Inherits: ^AntiGroundMissile
 	ReloadDelay: 60
-	Range: 5c0
 	MinRange: 1c256
-	Report: missile6.aud
 	Burst: 4
 	BurstDelay: 14
-	ValidTargets: Ground, Water
 	Projectile: Missile
 		Speed: 256
-		Arm: 2
-		Blockable: false
-		ContrailLength: 10
-		Inaccuracy: 128
-		Image: DRAGON
 		HorizontalRateOfTurn: 10
 		RangeLimit: 8c512
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 60
-		ValidTargets: Ground, Water
 		Versus:
 			None: 30
 			Wood: 90
 			Light: 90
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
-	Warhead@3Eff: CreateEffect
-		Explosions: med_explosion
-		ImpactSounds: kaboom25.aud
-		InvalidImpactTypes: Water
-	Warhead@4EffWater: CreateEffect
-		Explosions: med_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
+			Heavy: 100
+			Concrete: 100
 
 HellfireAA:
+	Inherits: ^AntiAirMissile
```

Comment:
> This also adds a minrange of half a cell, which is a good thing but again shouldn't go unmentioned.

This one isn't true, it has always had `MinRange: 1c256` (see 3 lines below our comments).